components/dashboard/SuggestedQuests.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect, useState } from "react";
+import Typography from "@components/UI/typography/typography";
+import { TEXT_TYPE } from "@constants/typography";
+import QuestStyles from "@styles/Home.module.css";
+import QuestCardCustomised from "./CustomisedQuestCard";
+import { getQuests } from "@services/apiService";
+import { QuestDocument } from "../../types/backTypes";
+
+const SuggestedQuests: React.FC = () => {
+    const [quests, setQuests] = useState<QuestDocument[]>([]);
+
+    useEffect(() => {
+        const fetchQuests = async () => {
+            const res = await getQuests();
+            if (res && res.onboarding) {
+                setQuests(res.onboarding.slice(0, 3));
+            }
+        };
+
+        fetchQuests();
+    }, []);
```

Comment:
Just a little optimization here:
As we use quests a lot in the website, it would involve a lot of requests if we had to fetch them everywhere we use them. Instead, we just fetch them once and store them in a shared state.
You can use 
```tsx
  const { quests } =
    useContext(QuestsContext);
```
to access it (and them simply keep the 3 firsts).
Tell me if it makes sense to you!

Reply:
@Marchand-Nicolas Yeah, that makes total sense. Does the new commit 08880e9fe2a5ce8e1641be2d96d5535bc9db9da0 solve the problem? I wasn't clear if you wanted me to display the first 3 quests from the entire array or just those from the onboarding category.